lib/datadog/tracing/contrib/http/instrumentation.rb

```diff
@@ -79,8 +80,10 @@ def annotate_span_with_request!(span, request, request_options)
 
               span.set_tag(Tracing::Metadata::Ext::TAG_COMPONENT, Ext::TAG_COMPONENT)
               span.set_tag(Tracing::Metadata::Ext::TAG_OPERATION, Ext::TAG_OPERATION_REQUEST)
-
-              span.set_tag(Tracing::Metadata::Ext::HTTP::TAG_URL, request.path)
+              span.set_tag(
+                Tracing::Metadata::Ext::HTTP::TAG_URL,
+                Contrib::Utils::Quantization::HTTP.url(request.path, { query: { exclude: :all } })
```

Comment:
This seems fine as a default, and you've added tests to make sure there aren't any query strings in other integrations.

Is it worth putting this same quantization line in those integrations as well? For consistency? Or is there a drawback?

Reply:
Currently, other http client integration do not offer settings for quantization, each of them leverage their public interface to provide data. 

I believe we should refactor and centraliz the qunatization mechanism when we offer the feature.